src/main/java/org/broadinstitute/hellbender/engine/VariantWalker.java

```diff
@@ -55,6 +55,14 @@ void initializeFeatures() {
         initializeDrivingVariants();
     }
 
+    @Override
+    public void pastInitialize() {
+        if ( hasIntervals() ) {
+            drivingVariants.setIntervalsForTraversal(intervalsForTraversal);
+        }
+    }
```

Comment:
Now that I've seen the rest of the code, I'm not sure that this new `pastInitialize()` method is a good idea. You can already override `onStartup()` to do this kind of post-engine-initialization initialization. Eg., this is how `ReadWalker` sets the intervals for its driving source of reads:

    @Override
    protected final void onStartup() {
        super.onStartup();

        if ( hasIntervals() ) {
            reads.setIntervalsForTraversal(intervalsForTraversal);
        }
    }

Recommend that you do the same for `VariantWalker`, and just remove this new `pastInitialize()` method to avoid cluttering the interface.